wg-releases/wg-qa/README.md

```diff
@@ -21,16 +21,23 @@ Oversees stability, quality, and testing criteria within Electron.
 
 ## Current Objective and Key Results
 
-**Objective:**
+**Objective 1:**
 
-Electron stable releases should be of high quality and easy for apps to upgrade.
+Early detection of Electron issues
```

Comment:
So using the guidance from my general comment I think we could make the objectives more concrete. For example (and this is just something I thought for a minute and definitely not what I'm proposing as the final objective):

```suggestion
All compatibility issues between new Electron versions and current apps are detected before releasing a new major version
```
 
This is highly inspirational, and it will take quite a few different things and time to achieve (or even get close to that), so we can keep adding key results to the list as we make progress.

Reply:
Major version issues* between new Electron versions and current apps are detected before releasing a new major version

*Includes: chromium, node, & electron issues